packages/client/manifest.json

```diff
@@ -5186,5 +5186,88 @@
         "suffix": "repeater"
       }
     ]
+  },
+  "rowexplorer": {
+    "block": true,
+    "name": "Row explorer block",
+    "icon": "PersonalizationField",
+    "size": {
+      "width": 100,
```

Comment:
We should increase the size here, as this block is usually pretty large. This key is used to render the size of the placeholder when dragging the component from the "new component" panel. I think we should use 600x400, same as the table block.

Also noticed chart block uses 100x100 and it should also use something bigger, so feel free to quickly update that while you're at it!